### PR TITLE
Work in progress adding recipe , working on selecting ingredients from list to add to recipe

### PR DESCRIPTION
**Pull request by pmatar28code**
Still work in progress, managed to get near user selecting ingredients, and passed them to the other Dialog for it to view what's added.
I thrower away a ton of code, I committed and then revert.
got a bit tired, but I think im on the correct path, sorry for the mess, still need to clean up.
never though we would need simple arrays for something  <-- they suck!!

### CODE REVIEW

File: app/src/main/java/com/example/foodmanchu/AddRecipeDialog.kt [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+
+        var mutableListFalse = mutableListOf<Boolean>()
+        for(ingredient in Repository.IngredientsList){
+            mutableListFalse.add(false)
+        }
+
+        // Initialize a boolean array of checked items
+        var arrayChecked:BooleanArray = mutableListFalse.map { false }.toBooleanArray()
+
+
```
Comment (pcutting) [inline app/src/main/java/com/example/foodmanchu/AddRecipeDialog.kt:61]: lots of spacing, compact it some.

File: app/src/main/java/com/example/foodmanchu/RecipesFragment.kt [hunk 2]
```diff
@@ -10,6 +10,10 @@ class RecipesFragment: Fragment(R.layout.recipes_fragment) {
         super.onCreate(savedInstanceState)
         val binding = RecipesFragmentBinding.bind(view)
 
+        binding.addRecipeFab.setOnClickListener {
+            AddRecipeDialog().show(parentFragmentManager,"Open Add Recipe")
```
Comment (pcutting) [inline app/src/main/java/com/example/foodmanchu/RecipesFragment.kt:14]: this sounds strange... no comment on it other than that.

**Review by pcutting — APPROVED** [review]
Nothing big.  Some changes if you want.

File: app/src/main/java/com/example/foodmanchu/RecipesFragment.kt (hunk 2 above)
Comment (pmatar28code) [reply app/src/main/java/com/example/foodmanchu/RecipesFragment.kt:14]: yeah this one is wierd , but it works, its a multi selection alert dialog, that's where ingredients are added to a recipe, it shows them in a textview then I build a string based on the textview.text and set the object parameter to that. as well as other stuff in the other dialog lol

**Pull request closed without merging**